src/contrib/mlir/lowerer.cpp

```diff
@@ -530,6 +530,102 @@ namespace
         return matchSuccess();
     }
 
+    REWRITER(NGConcatOp)
+    {
+        auto concat = cast<NGConcatOp>(op);
+        auto loc = concat.getLoc();
+
+        ScopedContext scope(rewriter, loc);
```

Comment:
This is used by LoopBuilders, maybe better to move to right before it is needed below.

Reply:
I can't seem to move this much lower without a segfault. Looks like something down in the bowels of MemRefView's ctor wants to refer to a ScopedContext.